homeassistant/components/bluesound/media_player.py

```diff
@@ -557,17 +552,17 @@ def _create_service_item(item):
     def state(self):
         """Return the state of the device."""
         if self._status is None:
-            return STATE_OFF
+            return MediaPlayerState.OFF
 
         if self.is_grouped and not self.is_master:
             return STATE_GROUPED
```

Comment:
Do we allow custom states for `media_player`?

Reply:
No, we only allow the documented states, in the enum.

Reply:
Thanks for confirming. I have adjusted to use `MediaPlayerState.IDLE` instead.